WordPress/Classes/Services/BlogSyncFacade.m

```diff
@@ -49,7 +49,7 @@ - (void)syncBlogWithUsername:(NSString *)username
     blog.password = password;
     blog.options = options;
     [[ContextManager sharedInstance] saveContext:context];
-    [blogService syncBlog:blog success:nil failure:nil];
+    [blogService syncBlog:blog];
```

Comment:
Since we already have the `options` I think we could skip the sync here as well.